src/scss/parse.js

```diff
@@ -4038,106 +4023,258 @@ function _checkUnicodeWildcard(i) {
 }
 
 /**
- * Check if token is part of URI (e.g. `url('/css/styles.css')`)
- * @param {Number} i Token's index number
- * @returns {Number} Length of URI
+ * Check if token is part of URI, e.g. `url('/css/styles.css')`
+ * @param {number} i Token's index number
+ * @returns {number} Length of URI
  */
 function checkUri(i) {
-  var start = i;
+  const start = i;
+  let token;
 
-  if (i >= tokensLength || tokens[i++].value !== 'url' ||
-      i >= tokensLength || tokens[i].type !== TokenType.LeftParenthesis)
-      return 0;
+  if (i >= tokensLength || tokens[i].value !== 'url') return 0;
 
-  return tokens[i].right - start + 1;
+  // Skip url
+  i++;
+
+  if (i >= tokensLength || tokens[i].type !== TokenType.LeftParenthesis)
+    return 0;
+
+  // Store the opening parenthesis token as we will reference it's `right`
+  // property to determine when the parentheses close
+  token = tokens[i];
```

Comment:
You don't reassign `token` anywhere later so you can move `let` from line 4032 here and rename variable to something like `leftParenthesis`